### ISSUE TITLE
Incorrect handling of the "elevation" feature by installers

### ISSUE THREAD

**Issue by pchemguy** [dokan-dev/dokany#903]
# Environment
 * Hardware: Virtual - VMware
 * Windows version: 7sp1
 * Processor architecture: x64 
 * Dokany version: 1.4.0.1000
 * Library type (Dokany/FUSE): 
 
# Check List
  - [+] I checked my issue [doesn't exist yet](https://github.com/dokan-dev/dokany/issues?utf8=%E2%9C%93&q=is%3Aissue)
  - [+] My issue is valid with mirror default sample and not specific to my user-mode driver implementation
  - [+] I can always reproduce the issue with the provided description below.
  - [+] I have updated Dokany to the [latest version](https://github.com/dokan-dev/dokany/releases) and have reboot my computer after.
  - [-] I tested one of the last [snapshot](https://github.com/dokan-dev/dokany/wiki/Build#user-snapshot) from appveyor CI

# Description
The installer is incorrectly handles the "elevation" feature. I always disable UAC, so if a program is run by and admin, it has admin privileges. As can be seen from the log, installation fails after the installer fails to elevate. This is wrong. At most, it should attempt elevation and proceed regardless of the result.

# Logs
[Dokan_Library_1.4.0.1000_Bundle_20200605001858.log](https://github.com/dokan-dev/dokany/files/4732785/Dokan_Library_1.4.0.1000_Bundle_20200605001858.log)

**Comment by Liryna** [comment]
Hi @pchemguy 

I tried to reproduce the issue by disabling UAC on my Win10/Win7 VM and running `DokanSetup.exe` but could not face the issue. Do you have more information for helping me reproducing the issue ?

It might even not be a elevation issue. Error 0x80070483 ERROR_NO_ASSOCIATION `No application is associated with the specified file for this operation.` in the log would be more like he has no idea how to open `C:\Windows\Temp\{C9DEB154-E81E-46AF-BFFB-536F9A9EC454}\.be\DokanSetup.exe` file ? I might be configuration system you have that produce the issue.

![image](https://user-images.githubusercontent.com/4311419/83850111-b7038d00-a710-11ea-9594-d07a5c6b9653.png)
![image](https://user-images.githubusercontent.com/4311419/83850407-28434000-a711-11ea-8437-2286dac39a8c.png)

**Comment by pchemguy** [comment]
Ok, I traced the source of the issue I experienced. Because I disable UAC, I also remove "runas" verb (HKEY_CLASSES_ROOT\exefile\shell\runas), which is part of the elevation infrastructure. Even if UAC is disabled, when elevation is requested, this verb is used. If missing, a file association error is thrown. While the caller should probably still ignore this error, I can now fix the issue for myself by restoring the runas verb.
